api_app/models/domain/airlock_resource.py

```diff
@@ -1,35 +1,11 @@
 from enum import Enum
-from typing import List
-from pydantic import Field
-from models.domain.azuretremodel import AzureTREModel
 from resources import strings
 
 
+# left for db migration
```

Comment:
Just wondering if it's worth moving this class closer to DB migration if it's the only place it's used in

Reply:
I deleted it for now, if migrations will be implemented I will add it again